app/models/api/v1/query_param.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# To be included in api controllers for handeling query params
+module Api
+  module V1
+    module QueryParam
+      def invalid_query_param(name, status, msg)
+        render status: status, json: json_api_error(msg, error_options:
```

Comment:
I'm still reusing `json_api_error` from base_controller. Could be an idea to extract all error handeling to a separate module, but felt a bit too much for now